pkg/agent/agent.go

```diff
@@ -47,6 +48,7 @@ import (
 	"antrea.io/antrea/pkg/ovs/ovsconfig"
 	"antrea.io/antrea/pkg/ovs/ovsctl"
 	"antrea.io/antrea/pkg/util/env"
+	antreaIPs "antrea.io/antrea/pkg/util/ip"
```

Comment:
Please follow the alias convention, no upper case, normally the alias would be `utilip`

Reply:
updated.